src/content/api/accounts.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Accounts
+description: Account model and related endpoints
+draft: true
+nav:
+  path: API
+  order: 7
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import CodeGroup from '../../components/CodeGroup.astro';
+
+An Account represents a permission boundary around Centrapay resources.
+Accounts can have [API Keys](/api/api-keys/) and [Account Memberships](/api/account-memberships/) which grant access to the resources.
```

Comment:
@MeganSteenkamp linkinator is failing because `/api/account-memberships/` is not a valid link yet. What do you think we should do here?

I think we should only validate links on the production site - this allows us to migrate each page with a single PR. Links will then be validated in the final PR which publishes every page.

The other option is to leave these links and come back to them for each page but I think this will take longer.

Reply:
@AndyClifford I think we can make them absolute links to pass the pipeline and then migrate them all at the end.

Reply:
I agree with Megan. This way we won't miss any links resulting in what we are trying to avoid with the check.

Reply:
I can see the ProseA custom component has logic to determine whether or not the link is for the api reference

`const isApiReferenceLink = props.href.startsWith('https://docs.centrapay.com/api');`

Let's leave the validation as is and just put in the full path. We can open a PR at the end to change this logic allowing our links to be relative.